index.js

```diff
@@ -73,17 +74,20 @@ const dropbox = {
   action,
   config: {
     path: {
-      title: 'Directory to save the files in',
+      title: 'Upload Directory',
+      description: 'The directory to save the files in.',
       type: 'string',
-      pattern: '^\/(.|[\r\n])*[^\/]$'
+      pattern: /^\/(.|[\r\n])*[^/]$/.source,
```

Comment:
Why are you allowing newlines here?

Reply:
I'm not sure where I got that regex from. I haven't look at this code in a long time. I mostly updated the auth stuff. I can take a look and make it a bit more strict